test/integration.spec.js

```diff
@@ -181,6 +187,31 @@ function testComponentProps (optionsToParse) {
   })
 }
 
+describe('component.props (es6 Array)', () => {
+  let component = {}
+
+  parser.parse(optionsForPropsArray)
+    .then((_component) => (component = _component))
+    .catch((err) => { throw err })
+
+  it('should list props from string array', () => {
+    const propsNames = component.props.map((item) => item.name)
```

Comment:
What about item.type? I think that `item.type` should be equal to `Any` in this case. What do you think?